annotation/tests/test_annotations.py

```diff
@@ -482,3 +588,491 @@ def test_construct_annotated_doc_db_error(
+                **page_revision_list_latest[1][annotated_doc_schema.user]
+            }
+        },
+    }
+    expected_pages[2][annotated_doc_schema.user]["page_id"] = (
+        annotated_doc_schema.pages["2"]
+    )
+
+    with patch(
+        "annotation.annotations.main.AnnotatedDocSchema.from_orm",
+        return_value=AnnotatedDocSchema(
+            revision=annotated_doc.revision,
+            user=annotated_doc.user,
+            pipeline=annotated_doc.pipeline,
+            date=annotated_doc.date,
+            file_id=annotated_doc.file_id,
+            job_id=annotated_doc.job_id,
+            pages=annotated_doc.pages,
+            validated=annotated_doc.validated,
+            failed_validation_pages=annotated_doc.failed_validation_pages,
+            tenant=annotated_doc.tenant,
+            task_id=annotated_doc.task_id,
+            similar_revisions=None,
+            categories=annotated_doc.categories,
+            links_json=annotated_doc.links_json,
+        ),
+    ) as mock_from_orm, patch(
+        "annotation.annotations.main.mark_latest_revision_validated_pages"
+    ) as mock_mark:
+        actual_pages = find_latest_revision_pages([annotated_doc], {1, 2})
+        mock_from_orm.assert_called_once_with(annotated_doc)
+        mock_mark.assert_called_once()
+        assert actual_pages == expected_pages
+
+
+def test_load_page(page_revision_list_all: Dict[int, List[PageRevision]]):
+    actual_loaded = []
+    expected_loaded = [
+        {
+            "page_num": 1,
+            "size": {"width": 0.0, "height": 0.0},
+            "objs": [],
+        }
+    ]
+    load_page(
+        Mock(),
+        actual_loaded,
+        bucket_name=TEST_TENANT,
+        page_num=1,
+        user_id=uuid.uuid4(),
+        page_revision=page_revision_list_all[1][0],
+        is_not_particular_revision_page=False,
+    )
+    assert actual_loaded == expected_loaded
+
+
+def test_load_page_particular_existing_revision(
+    page_revision_list_all: Dict[int, List[PageRevision]]
+):
+    page_revision_list_all[1][0]["page_id"] = "sha1"
+    page_path = (
+        f"annotation/{page_revision_list_all[1][0]['job_id']}/"
+        f"{page_revision_list_all[1][0]['file_id']}/"
+        f"{page_revision_list_all[1][0]['page_id']}.json"
+    )
+
+    actual_loaded = []
+    expected_loaded = [
+        {
+            "page_num": 1,
+            "size": {"width": 0.0, "height": 0.0},
+            "objs": [],
+            "revision": page_revision_list_all[1][0]["revision"],
+            "user_id": page_revision_list_all[1][0]["user_id"],
+            "pipeline": page_revision_list_all[1][0]["pipeline"],
+            "date": page_revision_list_all[1][0]["date"],
+            "is_validated": page_revision_list_all[1][0]["is_validated"],
+            "categories": page_revision_list_all[1][0]["categories"],
+        }
+    ]
+
+    with patch(
+        "annotation.annotations.main.S3_START_PATH", "annotation"
```

Comment:
Could we start using the named argument new in our patch() calls like patch('path.to.module', new='value')? It makes it immediately clear what the second param refers to.
```suggestion
        "annotation.annotations.main.S3_START_PATH", "annotation"
```
```suggestion
        "annotation.annotations.main.S3_START_PATH",new= "annotation"
```